CMakeLists.txt

```
@@ -39,6 +39,8 @@ option(PORTFFT_BUILD_TESTS "Whether to enable building tests" OFF)
 option(PORTFFT_BUILD_BENCHMARKS "Whether to enable building benchmarks" OFF)
 option(PORTFFT_VERIFY_BENCHMARKS "Whether to verify FFT results during benchmarking. Verifies in the first iteration only" OFF)
 option(PORTFFT_ENABLE_DOUBLE_BUILDS "Enable building tests and benchmarks using double precision" ON)
+option(PORTFFT_ENABLE_BUFFER_BUILDS "Enable building tests with buffers" ON)
+option(PORTFFT_ENABLE_OOP_BUILDS "Enable building testswith out-of-place configuration where an equivalent in-place configuration is supported" ON)
```

```suggestion
option(PORTFFT_ENABLE_OOP_BUILDS "Enable building tests with out-of-place configuration where an equivalent in-place configuration is supported" ON)
```
I don't think its worth the delays that will be caused by fixing this though.